### PR TITLE
additional option when deleting webapp

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**

### CODE REVIEW

**Comment by yaohaizh** [comment]
Is there any issue/features related to this PR?

**Comment by weidongxu-microsoft** [comment]
@yaohaizh
https://github.com/azure/azure-libraries-for-net/issues/466

File: src/ResourceManagement/AppService/WebAppDeleteOption.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Management.AppService.Fluent
+{
+    using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+
+    /// <summary>
+    /// Defines options for WebApp deletion.
+    /// </summary>
+    public class WebAppDeleteOption : IBeta
```
Comment (yaohaizh) [inline src/ResourceManagement/AppService/WebAppDeleteOption.cs]: Should be app service delete option

Comment (yaohaizh) [reply src/ResourceManagement/AppService/WebAppDeleteOption.cs]: Second though, should we remove this class to keep the interface consistent and simple to use. 

Provide default on the interface level.

File: src/ResourceManagement/AppService/ISupportWebAppDeleteOption.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+namespace Microsoft.Azure.Management.AppService.Fluent
+{
+    using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Provides access to deleting a resource from Azure, with additional option.
+    /// </summary>
+    public interface ISupportWebAppDeleteOption : IBeta
```
Comment (yaohaizh) [inline src/ResourceManagement/AppService/ISupportWebAppDeleteOption.cs]: ISupportAppServiceDelete

File: src/ResourceManagement/AppService/WebAppDeleteOption.cs (hunk 1 above)
Comment (weidongxu-microsoft) [reply src/ResourceManagement/AppService/WebAppDeleteOption.cs]: OK. Interface updated to use bool? .

**Pull request merged**